TimedSubdomainModifier (#27092)
idaholab/moose: framework/doc/content/source/userobjects/TimedElementSubdomainModifier.md

# TimedElementSubdomainModifier

## Overview

The `TimedElementSubdomainModifier` is a base class to be inherited by other user objects. It inherits

> from [CoupledVarThresholdElementSubdomainModifier.md] and additionally provides the ability to move elements

Reviewer: ```suggestion
from [ElementSubdomainModifier.md] and additionally provides the ability to change elements’ subdomains
```

at given times.

> This base class is inherited by

Reviewer: ```suggestion
```

> - [TimedSubdomainModifier.md]

Reviewer: ```suggestion
```

# TimedElementSubdomainModifier

!syntax description /UserObjects/TimedElementSubdomainModifier

## Overview

The `TimedElementSubdomainModifier` is a base class to be inherited by other user objects. It inherits 
from [CoupledVarThresholdElementSubdomainModifier.md] and additionally provides the ability to move elements

> at pre-given points in time.

Reviewer: "points" might be a little bit misleading here.
```suggestion
at given times.
```
